static/app/components/events/errorLevel.tsx

```diff
@@ -14,7 +14,7 @@ type Props = {
 };
 
 function ErrorLevel({className, level = 'unknown', size = '11px'}: Props) {
-  const levelLabel = tct('Level: [level]', {level: capitalize(level)});
+  const levelLabel = tct('Log Level: [level]', {level: capitalize(level)});
```

Comment:
We changed to `Level` from `Log Level` before because of the new issues types, but now that we only show for errors we can refer to log level again